ublox/src/parser.rs

```diff
@@ -237,19 +239,42 @@ impl<T: UnderlyingBuffer, P: UbxProtocol> Parser<T, P> {
             _phantom: PhantomData,
         }
     }
+
+    /// Appends `new_data` to the internal buffer and returns and iterator over the buffer
+    /// that will yield [UbxPackets, RtcmPackets, or NmeaPackets](AnyPacketRef) on demand.
+    pub fn consume_ubx_rtcm_nmea<'a>(
```

Comment:
I think we just want to delete the previous `ubx_rtcm` and simply have a single method here, this very one and only.
Don't you guys agree ? @andrei-ng

Reply:
I assume we can also avoid any API breakage then

Reply:
> I assume we can also avoid any API breakage then

I was thinking the same...